### DISCUSSION TITLE
Adding Options field in mongdb supported methods

### DISCUSSION THREAD

**Discussion by debasispanigrahi** [prisma/prisma#21931] [Q&A]
Currently some fields like where,skip, orderby etc are given in mongodb methods. Can we have another field options where we can use any of these fields https://www.mongodb.com/docs/manual/reference/command/aggregate/#command-fields. It is only for aggregate. But similar can be used for other methods.

Application: This will be helpful for collation 

issue: 2: Adding Regex field for string along with its options.
Currently we have only startsWith, contains, endsWith etc for string method. But no regex. but we can define two fields for regrex accepting string and its options.

**Comment by ludralph** [comment]
Hi @debasispanigrahi 👋 

Thank you for raising this question.

At the moment, `options` field is not available as a first class support. However, you should be able to use `options` field with the [aggregateRaw](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#aggregateraw) command in Prisma with MongoDB.

> issue: 2: Adding Regex field for string along with its options.

You can also achieve this using [findRaw](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#findraw) and `aggregateRaw` command. 

You can also open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and document your use case so our engineers can take a look if you want first class support for `field options` and `Regex field for string along with its options`.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
